mysql-test/suite/rocksdb.sys_vars/r/all_vars.result

```diff
@@ -1,15 +1,15 @@
 create table t1 (test_name text) engine=MyISAM;
 create table t2 (variable_name text) engine=MyISAM;
 load data infile "MYSQLTEST_VARDIR/tmp/rocksdb_sys_vars.all_vars.txt" into table t1;
-insert into t2 select variable_name from information_schema.global_variables where variable_name like "rocksdb_%";
-insert into t2 select variable_name from information_schema.session_variables where variable_name like "rocksdb_%";
+insert into t2 select variable_name from performance_schema.global_variables where variable_name like "rocksdb_%";
+insert into t2 select variable_name from performance_schema.session_variables where variable_name like "rocksdb_%";
 select variable_name as `There should be *no* long test name listed below:` from t2
 where length(variable_name) > 50;
 There should be *no* long test name listed below:
 select variable_name as `There should be *no* variables listed below:` from t2
 left join t1 on variable_name=test_name where test_name is null ORDER BY variable_name;
 There should be *no* variables listed below:
```

Comment:
LOL, I noticed that too and I don't think so...

I just sent one for them https://github.com/facebook/mysql-5.6/issues/575